ipatests/pytest_ipa/integration/expect.py

```diff
@@ -107,18 +107,44 @@ def send(self, s):
         logger.debug('Sending %r', s)
         return super().send(s)
 
-    def expect_list(self, pattern_list, *args, **kwargs):
+    def expect_list(
```

Comment:
If the signature of original function changes we will need to
a) somehow notice it
b) change the signature in this code.

Also duplicating and passing all parameters looks too verbose compared to `*args, **kwargs` approach.

Is not it better to temporary disable the failing rule in pylintrc or add a `# pylint: disable=W0222` comment on single line?